pages/utilites/aa_comp_calculator.md

```diff
@@ -18,6 +18,10 @@ Required arguments:
 Optional arguments:
 - `-o`, `--output <out_dir>` Path to user-defined output directory
   - Default: `./aa_comp_calculator_out_<M.D.Y>`
+- `--chimera` A .tsv containing a Unique ID, higher taxonomy, and lower taxonomy for each chimera within the input bootstrap files. The file contains three columns:
```

Comment:
Not bootstrap files but matrix right?

Reply:
No, bootstrap files. The script never operates on a matrix.

Reply:
Nevermind I did it backward. I was thinking this was bipartition examiner

Reply:
aa comp calculator does not operate on a matrix???